frame/election-provider-support/src/lib.rs

```diff
@@ -473,16 +473,9 @@ where
 	}
 }
 
-/// A utility trait for something to implement `ElectionDataProvider` in a sensible way.
-///
-/// This is generic over `AccountId` and it can represent a validator, a nominator, or any other
-/// entity.
-///
-/// The scores (see [`Self::Score`]) are ascending, the higher, the better.
-///
-/// Something that implements this trait will do a best-effort sort over ids, and thus can be
-/// used on the implementing side of [`ElectionDataProvider`].
-pub trait SortedListProvider<AccountId> {
+/// Basically a read-only part of the methods for [`SortedListProvider`] designed to be used in
+/// scenarios where the list is updated by another entity.
```

Comment:
could use a more contextual doc, and generally, can use some insight from https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html

Reply:
Those are so far just rough doc placeholders, I'll refine those as this gets closer to being merged.